scripts/generate-icons.mjs

```diff
@@ -6,6 +6,7 @@ import { pascalCase } from 'scule';
 import { XMLParser } from 'fast-xml-parser';
 
 const PREFIX = 'ri-';
+const LUCIDE_PREFIX = 'lucide-';
```

Comment:
Would it make sense to keep the prefix to something sorter like `lu`- or `li`

Reply:
hmm yeah I think so